cli/schedules/README.md

```diff
@@ -0,0 +1,43 @@
+## Working with Schedule in Azure Machine Learning CLI 2.0
+This repository contains an example `YAML` file for creating `schedule` using Azure Machine learning CLI 2.0. This directory includes:
+
+- Sample `YAML` files for creating a `schedule`. Currently the `schedule` can be used to associate with a `PipelineJob`, more job types & other resources will be supported in near future.
+
+
+- To create a schedule using any of the sample YAML files end with "-schedule.yml" provided in this directory, execute following command:
+```cli
+> az ml schedule create -f cron-schedule.yml
```

Comment:
need to generate ci workflows for these new schedule yaml

Reply:
yes, will do once we can have a workable CLI/SDK version